src/action_runner/action_runner.cpp

```diff
@@ -2,8 +2,6 @@
 #include <Windows.h>
 #include <shellapi.h>
 
-#include <string_view>
```

Comment:
how is this unused? search for `string_view` shows multiple matches. we shouldn't remove headers just because they're happened to be included transitively, since that could change.